scripts/dockerimages/scripts/network.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ ! -d "/sys/class/net/eth0" ]; then
+    echo "eth0 not found, ssh access cannot be provided!"
```

Comment:
The problem is larger then ssh. Should probably say network unavailable... should the script just exit? Since we won't get a dhcp address.